cli/crates/server/src/bridge/search.rs

```diff
@@ -56,20 +57,24 @@ impl<'a> Index<'a> {
                 searcher.search_backward_before(last as usize, &before.try_into()?)?
             }
         };
-        Ok(hits.map_id(|id| String::from_utf8(id).unwrap()))
+        Ok(hits.map_id(|id| {
+            let mut id = String::from_utf8(id).unwrap();
+            let _ = id.drain(..(id.len() - 26));
```

Comment:
A comment here might be good?  Looks quite weird, cannot immediately tell why we're doing this

Reply:
just removing the prefix: "post#<ulid>". Didn't want to make too many assumptions, but expecting an ulid at the end is one already. Adding a comment